cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java

```diff
@@ -122,6 +122,8 @@ public void handle(String target, HttpServletRequest request, HttpServletRespons
                         LogHandlers.getLoggers(request, response, configurationPath);
                     } else if (uri.startsWith("/v1/admin/log/appenders")) {
                         LogHandlers.getAppenders(request, response, configurationPath);
+                    } else if (uri.startsWith("/v1/admin/metrics") || uri.startsWith("/admin/metrics")) {
```

Comment:
Not sure if the `/v1` case should be included... Let's discuss by skype (I need to understand a little bit what `/v1` means for Cygnus).